### ISSUE TITLE
Pawn promotion animation

### ISSUE THREAD

**Issue by Meelo** [Meelo/NALCG#19]
When pawn is promoted from Pawn to Rook/Knight/Bishop/Queen, some sort of promotion animation should appear.

**Comment by Meelo** [comment]
Currently nice to have, since logic isn't implemented yet. Blocked by #18

**Comment by Dzarg** [comment]
Done
